<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostMulticast</class>
 <widget class="QDialog" name="HostMulticast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host Multicast</string>
  </property>
  <widget class="QListWidget" name="playlistWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>241</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="localLibraryList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>401</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Library</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
